### PR TITLE
Add VSCode related implementation for supporting Kubernetes.toml

### PR DESCRIPTION
**Pull request by xlight05**
## Purpose
$Subject.
This PR adds toml activation logic to vscode. 
The language server related changes are introduced in ballerina-platform/ballerina-lang#27869.
Additionally this implementation provides a command to create a sample Kubernetes.toml file in the project directory.

## Fixes
Resolves https://github.com/ballerina-platform/plugin-vscode/pull/135

### CODE REVIEW

File: src/project/cli-cmds/k8s.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+`
+
+export function activateK8sCommand() {
+    const reporter = ballerinaExtInstance.telemetryReporter;
+
+    // register create kubernetes.toml command handler
+    commands.registerCommand('ballerina.create.k8s', async () => {
+        try {
+            reporter.sendTelemetryEvent(TM_EVENT_CREATE_K8S, { component: CMP_K8S });
+            const currentProject = await getCurrentBallerinaProject();
```
Comment (prabushi) [inline src/project/cli-cmds/k8s.ts:62]: This command doesn't run if that particular toml file is open in the currently active editor. Is this the expected behavior?

File: package.json [hunk 2]
```diff
@@ -39,6 +39,16 @@
                     ".bal"
                 ],
                 "configuration": "./grammar/ballerina.configuration.json"
+            },
+            {
+                "id": "toml",
```
Comment (prabushi) [inline package.json:44]: Do we need to language contributor, configurationDefaults, and the toml scheme if we use the currently open bal project to get its toml file?

File: src/project/cli-cmds/k8s.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            if (currentProject.path) {
+                let k8sTomlPath = currentProject.path + CLOUD_CONFIG_FILE_NAME
+                if (!fs.existsSync(k8sTomlPath)) {
+                    fs.writeFile(k8sTomlPath, KUBERNETES_TOML_DEFAULT_CONTENT, (err) => {
```
Comment (prabushi) [inline src/project/cli-cmds/k8s.ts:66]: Once the command is executed there'll be no indication of what happens to the user. Shall we add a note to the output channel upon file creation? Can use someting similar to [this](https://github.com/prabushi/plugin-vscode/blob/master/src/project/cli-cmds/cmd-runner.ts#L10)

**Review by prabushi — APPROVED** [review]

**Review by nadeeshaan — APPROVED** [review]
LGTM +1 to proceed to merging

**Pull request merged**